give.php

```diff
@@ -234,6 +234,8 @@ public function boot()
 
         add_action('plugins_loaded', [$this, 'init'], 0);
 
+        add_action('admin_enqueue_scripts', [$this, 'enqueueAdminScripts'], -1);
+
```

Comment:
@kjohnson can we move this into a service provider?  I'd like to avoid adding to the main give file since we have the service providers for all our entry points.

Reply:
@jonwaldstein resolved, see https://github.com/impress-org/givewp/pull/6632/commits/a2a76fbe10ded6b01f0515f210dcf2e8125b8afb.